yarn.lock

```diff
@@ -10294,36 +10268,11 @@ minimatch@3.0.4, minimatch@^3.0.0, minimatch@^3.0.2, minimatch@^3.0.3, minimatch
   dependencies:
     brace-expansion "^1.1.7"
 
-minimist@0.0.5:
-  version "0.0.5"
-  resolved "https://registry.yarnpkg.com/minimist/-/minimist-0.0.5.tgz#d7aa327bcecf518f9106ac6b8f003fa3bcea8566"
-  integrity sha1-16oye87PUY+RBqxrjwA/o7zqhWY=
-
-minimist@0.0.8:
-  version "0.0.8"
-  resolved "https://registry.yarnpkg.com/minimist/-/minimist-0.0.8.tgz#857fcabfc3397d2625b8228262e86aa7a011b05d"
-  integrity sha1-hX/Kv8M5fSYluCKCYuhqp6ARsF0=
-
-minimist@1.2.0:
-  version "1.2.0"
-  resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.0.tgz#a35008b20f41383eec1fb914f4cd5df79a264284"
-  integrity sha1-o1AIsg9BOD7sH7kU9M1d95omQoQ=
-
-minimist@^1.1.1, minimist@^1.1.3, minimist@^1.2.0:
-  version "1.2.2"
-  resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.2.tgz#b00a00230a1108c48c169e69a291aafda3aacd63"
-  integrity sha512-rIqbOrKb8GJmx/5bc2M0QchhUouMXSpd1RTclXsB41JdL+VtnojfaJR+h7F9k18/4kHUsBFgk80Uk+q569vjPA==
-
-minimist@^1.2.5:
+minimist@0.0.5, minimist@0.0.8, minimist@1.2.0, "minimist@>=0.2.1 <1.0.0 || >=1.2.3", minimist@^1.1.1, minimist@^1.1.3, minimist@^1.2.0, minimist@^1.2.5:
```

Comment:
This change illustrates what I mean. I don't think the version specification works as we expect it to. You have set the resolution to be `>=0.2.1 <1.0.0 || >=1.2.3`, but as you can see here, it just uses the last part of the string and resolves every requirement to `1.2.5`. This I believe is because this version resolution description is just an RFC and [not yet implemented](https://github.com/yarnpkg/rfcs/blob/master/implemented/0000-selective-versions-resolutions.md#mapping-version-specifications). 

What if we see what package uses which minimist version and specify its resolution explicitly, like, for example now we have:
```
=> Found "@mapbox/geojson-rewind#minimist@1.2.0"
info This module exists because "mapbox-gl#@mapbox#geojson-rewind" depends on it.
```

What if we specify the exact dependency predecessor in the resolution like this:
```
"resolutions": {
		"**/geojson-rewind/minimist": ">=1.2.3",
```

Reply:
Thanks! I followed your recommendation and was able to resolve all your concerns mentioned above: bc2bd0e